addOns/postman/src/main/java/org/zaproxy/addon/postman/PostmanParser.java

```diff
@@ -75,15 +96,216 @@ public void importFromUrl(final String url) throws IllegalArgumentException, IOE
+
+        HttpMessage httpMessage;
+        try {
+            String rawUrl = url.getRaw();
+            httpMessage = new HttpMessage(new URI(rawUrl, false));
+        } catch (URIException | HttpMalformedHeaderException | NullPointerException e) {
+            return null;
+        }
+
+        httpMessage.getRequestHeader().setMethod(request.getMethod());
+
+        for (KeyValueData header : request.getHeader()) {
+            if (!header.isDisabled()) {
+                httpMessage.getRequestHeader().setHeader(header.getKey(), header.getValue());
+            }
+        }
+
+        Body body = request.getBody();
+
+        if (body.isDisabled()) {
+            return httpMessage;
+        }
+
+        String mode = body.getMode();
+
+        String bodyContent = "";
+        String contentType = "";
+
+        if (mode == null || body == null) {
+            return httpMessage;
+        }
+        if (mode.equals("raw")) {
+            Map<String, String> contentTypeMap =
+                    Map.of(
+                            "html", "text/html",
+                            "javascript", "application/javascript",
+                            "json", "application/json",
+                            "xml", "application/xml");
+
+            contentType = "text/plain";
+
+            if (body.getOptions() != null && body.getOptions().getRaw() != null) {
+                String language = body.getOptions().getRaw().getLanguage();
+
+                if (language != null) {
+                    contentType = contentTypeMap.getOrDefault(language.toLowerCase(), "text/html");
+                }
+            }
+
+            bodyContent = body.getRaw();
+        } else if (mode.equals("urlencoded")) {
+            contentType = "application/x-www-form-urlencoded";
+
+            StringBuilder urlencodedBodySB = new StringBuilder();
+
+            for (KeyValueData data : body.getUrlencoded()) {
+                if (!data.isDisabled()) {
+                    if (urlencodedBodySB.length() > 0) {
+                        urlencodedBodySB.append("&");
+                    }
+                    try {
+                        urlencodedBodySB
+                                .append(
+                                        URLEncoder.encode(
+                                                data.getKey(), StandardCharsets.UTF_8.name()))
+                                .append("=")
+                                .append(
+                                        URLEncoder.encode(
+                                                data.getValue(), StandardCharsets.UTF_8.name()));
+                    } catch (UnsupportedEncodingException e) {
+                    }
+                }
+            }
+
+            bodyContent = urlencodedBodySB.toString();
+        } else if (mode.equals("formdata")) {
+            String boundary = "----" + System.currentTimeMillis();
+
+            contentType = "multipart/form-data; boundary=" + boundary;
```

Comment:
Could use `HttpHeader` constant for the first part.

Reply:
There is no const for `multipart/form-data`

Reply:
There is (https://github.com/zaproxy/zaproxy/blob/76893768eb9c891c7e1c9e0e2aaaeb6429386914/zap/src/main/java/org/parosproxy/paros/network/HttpHeader.java#L102) but it was only added recently, it will have to be done for/after 2.14.